endpoints/repairs.md

```diff
@@ -1,8 +1,150 @@
 # Repairs
 
+- [Get repairs](#get-repairs)
 - [Create repairs](#create-repairs)
 - [Create repair documents](#create-repair-documents)
 
+## Get repairs
+
+`GET /public/v1/repairs` retrieves a [paginated list](../readme.md/#pagination) of repairs.
+
+Please note:
+
+- Optional [filtering](../readme.md/#filtering) is available for this endpoint if needed.
+
+<details>
+<summary>Filter attributes names</summary>
+<br>
+- id
+- vehicle
+- fault
+- fleet_number
+- registration
+- repair_number
+- reported_to
+- reported_by
+- date_reported
+- job_number
+- defect
+- defect_cleared_by_date
+- defect_cleared_by
+- odometer_open
+- hours_open
+- closed
+- date_closed
+- hours_closed
+- odometer_closed
+- purchase_order
+- location
+- created_at
+- updated_at
+- site
+- repair_items.repair_types
+- repairers.name
+- repairers.code
+- repair_items.assigned_user_id
+- repair_items.general_ledger_code_id
+</details>
+
+### Request
+
+```
+URL: https://api.gearbox.com.au/public/v1/repairs
+Method: GET
+Authorization: Bearer $ACCESS_TOKEN
+```
+
+### 200 - Successful response
+
+```JSON
+{"repairs": [
```

Comment:
This sample response will need to be updated to match the actual response from the API